systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificIsolatedST.java

```diff
@@ -193,6 +207,97 @@ void testLoadBalancerSourceRanges(ExtensionContext extensionContext) {
+
+        // reference Cluster Operator service account in RoleBindings
+        strimziClusterOperator020Namespaced.getSubjects().stream().findFirst().get().setNamespace(clusterOperator.getDeploymentNamespace());
+        strimziClusterOperator022LeaderElection.getSubjects().stream().findFirst().get().setNamespace(clusterOperator.getDeploymentNamespace());
+
+        final List<RoleBinding> roleBindings = Arrays.asList(
+                strimziClusterOperator020Namespaced,
+                strimziClusterOperator022LeaderElection
+        );
+
+        // ---- c) defining ClusterRoleBindings
+        final List<ClusterRoleBinding> clusterRoleBindings = Arrays.asList(
+            ClusterRoleBindingTemplates.getClusterOperatorWatchedCrb(clusterOperator.getClusterOperatorName(), clusterOperator.getDeploymentNamespace()),
+            ClusterRoleBindingTemplates.getClusterOperatorEntityOperatorCrb(clusterOperator.getClusterOperatorName(), clusterOperator.getDeploymentNamespace())
+        );
+
+        clusterOperator.unInstall();
+        // create namespace, where we will be able to deploy Custom Resources
+        cluster.createNamespace(CollectorElement.createCollectorElement(extensionContext.getRequiredTestClass().getName(), extensionContext.getRequiredTestMethod().getName()), namespaceWhereCreationOfCustomResourcesIsApproved);
+        clusterOperator = clusterOperator.defaultInstallation()
+            .withWatchingNamespaces(Constants.WATCH_ALL_NAMESPACES)
+            // use our pre-defined Roles
+            .withRoles(roles)
+            // use our pre-defined RoleBindings
+            .withRoleBindings(roleBindings)
+            // use our pre-defined ClusterRoles
+            .withClusterRoles(clusterRoles)
+            // use our pre-defined ClusterRoleBindings
+            .withClusterRoleBindings(clusterRoleBindings)
+            .createInstallation()
+            .runBundleInstallation();
+
+        // implicit verification that a user is able to deploy Kafka cluster in namespace <example-1>, where we are allowed
+        // to create Custom Resources because of `*-namespaced Role`
+        resourceManager.createResource(extensionContext, false, KafkaTemplates.kafkaEphemeral(testStorage.getClusterName(), 3)
+            .editMetadata()
+            // this should work
+            .withNamespace(namespaceWhereCreationOfCustomResourcesIsApproved)
+            .endMetadata()
+            .build());
+
+        resourceManager.createResource(extensionContext, false, KafkaTemplates.kafkaEphemeral(testStorage.getClusterName(), 3)
+            .editMetadata()
+                // this should not work
+                .withNamespace(clusterOperator.getDeploymentNamespace())
+            .endMetadata()
+            .build());
+
+        // verify that in `infra-namespace` we are not able to deploy Kafka cluster
+        KafkaUtils.waitForKafkaStatusUpdate(clusterOperator.getDeploymentNamespace(), testStorage.getClusterName());
```

Comment:
Would it be more stable if we go with `waitUntilKafkaStatusConditionContainsMessage()`?
I'm afraid that there can be few moments when we don't get desired failure message